lib/phoenix_test/playwright.ex

```diff
@@ -387,15 +390,18 @@ defmodule PhoenixTest.Playwright do
   end
 
   @doc false
-  def retry(fun, backoff_ms \\ [100, 250, 500, timeout()])
-  def retry(fun, []), do: fun.()
+  def retry(fun, current_sleep \\ 0, max_sleep \\ timeout())
```

Comment:
Just to note: `retry` is also used for
- `assert_has("title", [opts])`
- `assert_path`
- `refute_path`

Same reasoning applies to those, too, though - most people probably prefer tests to complete asap rather than having increasing backoff.